engine/hatchery/openstack/openstack.go

```diff
@@ -804,6 +802,10 @@ func getServers(endpoint, token string) ([]Server, error) {
 		if !worker {
 			continue
 		}
+		workerHatcheryName, ok := s.Metadata["hatcheryName"]
+		if !ok || workerHatcheryName != h.Hatchery().Name {
```

Comment:
we should delete server with medata worker and without hatcheryName metadata to be able to delete old server just after put in production